_pages/en/it-picture-long-term.md

```diff
@@ -6,12 +6,75 @@ In the following sections, we're trying to take the same end state and create "v
+
+An example of the repercussions of such an implementation can be found [here](https://github.com/sara-sabr/ITStrategy/blob/master/wip-tec/en/human-development-life-cycle.md).
+
+In a service based environment, success is not measured by the number of users your service attracts, but rather the number of users amassed by the services that leverage your service. 
+
+For more details on it's implementation, please refer to the [Business section](##Businesses)
+
+### Modern Service Delivery 
+
+When was the last time you called the customer service of your favourite digital service? 
+
+Within 10 years ESDC should cease to inflict it's layering, processes, and complexities, onto it's users -- the citizens.
+As a government entity we are responsible to ensure the ease of use from the entirety of Canada's citizenry. 
+
+For those comfortable with consuming services digitally, which is becoming increasingly large majority of the population, physical interaction with government offices should be an option -- not a necessity.
+For regions which are still needing of attention to onboard digitally, different methods of service interaction need be available.
+Cost savings from automation and process improvement should be passed on to provide increased attention to areas where service uptake is lagging (*To-do of areas lacking use of services needed*)
+
+*Mode 1* 
+- Group: The growing number of citizens comfortable with an entirely digital 
+- Action: Develop digital services
+*Mode 2*
+- Group: Those unable or unwilling to use digital services in densely populated areas
+- Action: Implement initiatives to onboard users onto digital services 
+*Mode 3* 
+- Group: Those in areas lacking sufficient, or convenient access, to digital services 
+- Action: Maintain legacy centres, with a focus on digital education alongside offering services
+
+#### Emphasis on Automation
+
+> “Automatic enrolment reduces the burden on many seniors by enabling them to receive their Old Age Security pension and Guaranteed Income Supplement at age 65 without ever having to apply for them. We continue our efforts to modernize the Old Age Security and Guaranteed Income Supplement services to meet the needs of Canadians.”
+> > ––The Honourable Jean-Yves Duclos, Minister of Families, Children and Social Development
+
+Automation should ease the use of services, not increase the complexity. Through automation and integration service enrollment should be automated thereby simplifying the process for all. 
+
 ## Businesses
 
+- Able to find and integrate with all ESDC services
+
 ## Non-profits
 
-## IITB
+- Able to find and integrate with all ESDC services
+
+## IITB-ESDC
+
+- IITB is the digital, IM and IT advisor for ESDC and part of major business decisions
+- IITB and ESDC collaborate seamlessly to enable employees and deliver services
+- 90% of employees are equipped with mobile devices allowing them to work from anywhere
```

Comment:
I assume the remaining 10% would include employees in Service Centres still existing after the transition. Any other type of employees? We'll have to justify where the 90% figure comes from but that can be later.

Reply:
If we want to be safe for now, maybe change to "a majority" or "employees who can benefit from working from anywhere are equipped with mobile devices by default"